src/test/java/gim/model/ExerciseTrackerTest.java

```diff
@@ -64,10 +64,10 @@ public void hasExercise_exerciseWithSameIdentityFieldsInExerciseTracker_returnsT
         assertTrue(exerciseTracker.hasExercise(editedAlice));
     }
 
-    @Test
-    public void getExerciseList_modifyList_throwsUnsupportedOperationException() {
-        assertThrows(UnsupportedOperationException.class, () -> exerciseTracker.getExerciseList().remove(0));
-    }
+    //    @Test
```

Comment:
Is this used? As with several other unused but commented out tests. Might be better to take it out.